src/dlgConnectionProfiles.cpp

```diff
@@ -108,6 +110,7 @@ dlgConnectionProfiles::dlgConnectionProfiles(QWidget * parent)
         QIcon icon_new(QIcon::fromTheme(QStringLiteral("document-new"), QIcon(QStringLiteral(":/icons/document-new.png"))));
         QIcon icon_connect(QIcon::fromTheme(QStringLiteral("dialog-ok-apply"), QIcon(QStringLiteral(":/icons/dialog-ok-apply.png"))));
 
+        offline_button->setIcon(QIcon(QStringLiteral(":/icons/mudlet_editor.png")));
```

Comment:
I figured it was either the wrench or the hard hat, since this would presumably be used to edit or test your profile or scripts without connecting.
It doesn't matter to me, as my desktop environment doesn't show icons in this context.
We could hold a vote on discord if you like.

Reply:
Okay - I'll save it for a separate PR where I sneak in an implementation that causes a redesign of all the icons - __*muahahaha*__ :smiling_imp: